astrokat/observe_main.py

```diff
@@ -626,18 +631,16 @@ def main(args):
     # implementation, and this default setting then removed.
     opts.nd_params = 'off'
 
-    # get correlator settings from config files
-    args_ = None
-    if args:
-        import argparse
-        parser = argparse.ArgumentParser()
-        for arg in args:
-            # catch other hidden arguments such as correlator settings
-            if len(arg.split('=')[1]) > 1:
-                arg = "{}='{}'".format(arg.split('=')[0], arg.split('=')[1])
-            if arg.startswith(("-", "--")):
-                parser.add_argument(arg)
-        args_ = parser.parse_args(args)
+    # astrokat does not support observational command line parameters
+    # all observational parameters must be in the YAML file
+    # command line arguments will be dropped to unknown arguments
+    unknown_args = [arg for arg in args if '--' in arg]
+    if '--horizon' in unknown_args:
```

Comment:
The instruction sets often use the form `--horizon=20`, so this won't find it.  Could loop through all unknown and check if any start with `--horizon`.

Reply:
done

Reply:
if any('horizon' in arg for arg in unknown_args):